sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsClientContext.java

```diff
@@ -53,6 +54,15 @@ public void serialize(DiagnosticsClientConfig clientConfig, JsonGenerator genera
                 generator.writeStringField("machineId", ClientTelemetry.getMachineId(clientConfig));
                 generator.writeStringField("connectionMode", clientConfig.getConnectionMode().toString());
                 generator.writeNumberField("numberOfClients", clientConfig.getActiveClientsCount());
+                generator.writeObjectFieldStart("clientEndpoints");
+                for (String key: clientConfig.clientMap.keySet()) {
```

Comment:
Instead of iterating on keySet(), we can iterate on `entry`. Better readability.